config/config-example.js

```diff
@@ -73,10 +75,17 @@ exports.reportjoins = true;
 //   getting more than 160 or so users.
 exports.reportbattles = true;
 
-// report joins and leaves in battle - shows messages like "<USERNAME> joined" in battle
-// Turn this off on large tournament servers where battles get a lot of joins and leaves.
+// report joins and leaves in battle - shows messages like "<USERNAME> joined"
+//   This feature can clutter battle logs - turn this off if your server hosts
+//   battles with more than 30 or so spectators (such as big tournament battles).
```

Comment:
This should really not be done this way; you should detect how many users are in the battle and use that to determine whether or not to report battlejoins.